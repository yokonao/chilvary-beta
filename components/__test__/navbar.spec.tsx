import * as React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../navbar';

test('AppTitle', () => {
  const component = renderer.create(<Navbar />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});