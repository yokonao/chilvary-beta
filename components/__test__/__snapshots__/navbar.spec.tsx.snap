// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppTitle 1`] = `
<div
  className="navbar is-dark is-bold"
>
  <div
    className="navbar-menu"
  >
    <div
      className="navbar-start"
    >
      <div
        className="navbar-item"
      >
        <div
          className="button is-dark is-inverted"
        >
          <div>
            <a
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              <span
                className="icon has-text-dark"
              >
                <i
                  className="fas fa-lg fa-home"
                />
              </span>
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
